client/rest-high-level/src/main/java/org/elasticsearch/client/ilm/LifecyclePolicy.java

```diff
@@ -52,7 +52,18 @@
             PHASES_FIELD
         );
 
-        ALLOWED_ACTIONS.put("hot", Sets.newHashSet(UnfollowAction.NAME, SetPriorityAction.NAME, RolloverAction.NAME));
+        ALLOWED_ACTIONS.put(
+            "hot",
+            Sets.newHashSet(
+                UnfollowAction.NAME,
+                SetPriorityAction.NAME,
+                RolloverAction.NAME,
+                ReadOnlyAction.NAME,
+                ShrinkAction.NAME,
+                ForceMergeAction.NAME,
+                SearchableSnapshotAction.NAME
```

Comment:
I assume rollup is intentionally not here since it's experimental and not currently in the HLRC right?

Reply:
That's correct